packages/core/src/components/carousel/carousel-item/stories/carousel-item.react.stories.mdx

```diff
@@ -0,0 +1,12 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Carousel/Carousel Item" />
+# Carousel Item
+
+Wrapper of Swiper Slide, simplified and with UI to match our Style Guide. Read the [Swiper documentation](https://swiperjs.com/element) for any issue or for more information. Read more about the need of this component and its behavior at [ADR 0013](./?path=/docs/docs-architecture-decision-records-adr-0013-why-using-a-virtual-carousel-item--docs).
```

Comment:
(question) maybe could we shared it to avoid keep the same texto?

Reply:
Totally, done here @felipefialho [f4b6897](https://github.com/juntossomosmais/atomium/pull/265/commits/f4b689756c47e2cb8d60c23948089c4d7617af22)